inc/metabox/class-astra-meta-boxes.php

```diff
@@ -264,12 +276,37 @@ public function markup_meta_box( $post ) {
 				<div class="disable-section-meta">
 					<?php do_action( 'astra_meta_box_markup_disable_sections_before', $meta ); ?>
 
+					<?php if ( $show_meta_field && Astra_Builder_Helper::is_row_empty( 'above', 'header', 'desktop' ) ) { ?>
```

Comment:
```suggestion
					<?php if ( $show_meta_field && Astra_Builder_Helper::is_row_empty( 'above', 'header', 'desktop' ) ) : ?>
```

I believe this is a preferred way of writing the conditions inside of markup?

Reply:
@Nikschavan Yes, replaced `if {}` conditions with `if :`